test/setupVitest.ts

```diff
@@ -0,0 +1,96 @@
+import { beforeAll, afterAll } from 'vitest';
+import 'test/utils/addChaiAssertions';
+import 'test/utils/setupPickers';
+import 'test/utils/licenseRelease';
+import { generateTestLicenseKey, setupTestLicenseKey } from 'test/utils/testLicense';
+import { configure } from '@mui/internal-test-utils';
+import { config } from 'react-transition-group';
+
+import sinon from 'sinon';
+import { unstable_resetCleanupTracking as unstable_resetCleanupTrackingDataGrid } from '@mui/x-data-grid';
+import { unstable_resetCleanupTracking as unstable_resetCleanupTrackingDataGridPro } from '@mui/x-data-grid-pro';
+import { unstable_resetCleanupTracking as unstable_resetCleanupTrackingTreeView } from '@mui/x-tree-view';
+import { unstable_cleanupDOM as unstable_cleanupDOMCharts } from '@mui/x-charts/internals';
+import failOnConsole from 'vitest-fail-on-console';
+import { clearWarningsCache } from '@mui/x-internals/warning';
+import { isJSDOM } from './utils/skipIf';
+
+// Core's setupVitest is causing issues with the test setup
```

Comment:
What kind of issues?
Are there any actions to be taken to enable the reuse of the core config? 🤔

Reply:
I don't remember anymore. This was still when @Janpot was testing the setup in core.